engine/api/application_deployment.go

```diff
@@ -87,10 +94,11 @@ func (api *API) postApplicationDeploymentStrategyConfigHandler() service.Handler
 		oldPfConfig.MergeWith(pfConfig)
 
 		if err := application.SetDeploymentStrategy(tx, proj.ID, app.ID, pf.Model.ID, pfName, oldPfConfig); err != nil {
-			return sdk.WrapError(err, "postApplicationDeploymentStrategyConfigHandler")
+			return sdk.WithStack(err)
```

Comment:
I guess `application.SetDeploymentStrategy` is already returning (or should) an errorWithStack